app/src/ui/changes/changed-file.tsx

```diff
@@ -105,6 +107,19 @@ export class ChangedFile extends React.Component<IChangedFileProps, void> {
       })
     }
 
+    if (this.props.status !== FileStatus.Deleted) {
+      items.push(
+        { type: 'separator' },
+        {
+          label: 'Open in external editor',
```

Comment:
SourceTree just has 'Open' which actually feels pretty natural. Depending on what the file is it might not be an editor.

![image](https://cloud.githubusercontent.com/assets/2047188/26300041/324dc9ae-3e9a-11e7-8656-20076f00919c.png)